sw/device/lib/dif/dif_rv_core_ibex.h

```diff
@@ -280,6 +280,19 @@ dif_result_t dif_rv_core_ibex_get_rnd_status(
     const dif_rv_core_ibex_t *rv_core_ibex,
     dif_rv_core_ibex_rnd_status_t *status);
 
+typedef uint32_t dif_rv_core_ibex_fpga_info_t;
+/**
+ * Read the FPGA build timestamp info. This function only return valid data for
+ * fpga, for all other variants it is simply 0.
```

Comment:
Nits:
"return" -> "returns".
"for fpga." -> "for fpga variants."? (or environments, or whatever the most appropriate name is)

Reply:
Thanks Luis, done.